src/office365.cr

```diff
@@ -538,6 +538,7 @@ class Office365::Event
       title: @subject,
       body: @body.try &.content,
       attendees: attendees,
+      hide_attendees: @hide_attendees,
```

Comment:
This file also needs the mapping going the other way
place event => office365 event

Need to update the `private def event_params(event)` function

Reply:
ah, I see. I was looking for a `from_` or `to_` for the other way.